### ISSUE TITLE
Which values to use from realSFS folded spectrum?

### ISSUE THREAD

**Issue by mrescalona** [xiaoming-liu/stairway-plot-v2#18]
I'm using angsd's realSFS -fold 1 to create a folded SFS from my VCF file with 25 diploid samples.

The function returns 51 bins ranging from 0-50 alleles, with all values from 0-25 being non-zero and the remainder being zeroes. In a unfolded spectrum, I'd remove the first and last bins (monomorphic) and have 2n-1 (assuming all 51 bins are non-zero).

For the folded SFS, I have n/2+1 bins, ranging from 0 to 25. Which values should I keep, in this case?

Here is the folded SFS provided by angsd
22.779504 5737.983351 2271.759531 890.704561 602.396447 453.041958 378.123061 277.515939 310.205540 177.239056 232.210910 212.339813 174.853681 142.583014 126.940380 193.427710 103.595297 161.210691 113.174401 180.349763 112.341701 113.933613 110.531206 104.876343 124.583636 44.298895 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000 0.000000

**Comment by vanessaiza** [comment]
Hello, can you solve this?

**Comment by Rilquer** [comment]
Hello @vanessaiza and @mrescalona 

Reading through the manual and some other issues raised in this github, I believe we should remove the first bin (monorphic) for folded SFS, which gives us n/2 bins (n being the total number of **sequences**, which is twice the number of diploid invididuals sequenced). In the example above, that would give you 25 bins.

**Comment by Charlotte7654** [comment]
Hi, I have the same question. Thank you @Rilquer. I did the same, which that was the only way my plot made biological sense. 

@xiaoming-liu can you please confirm that we should remove the first bin (monorphic) for folded SFS, which gives us n/2 bins? It is very important to know that we are removing the correct number.

Thanks so much in advance for all your support!

**Comment by xiaoming-liu** [comment]
Yes, the SFS bins (folded) begin from 1 to n/2, of total n/2 bins. L is the
sum of all bins plus the monomorphic site count.

**Comment by Charlotte7654** [comment]
Thank you for the very quick response, Xiaoming!

**Comment by vanessaiza** [comment]
@xiaoming-liu Hi, so if my bin for folds is these: 109.9347447456476 130.4247795779732 109.7134440879069 79.7986021587142 57.33235049610008 43.83987352617628 36.60351143721994 33.56941738265945 33.97657594388816 37.2134755892125 41.66932957787554 44.86468165542625 0 0 0 0 0 0 0 0 0 0 0 0 0

The parameter L would be = 805.8329133

805.8329133 = adding each value (109.9347447456476 + 130.4247795779732 + 109.7134440879069 + 79.7986021587142 + 57.33235049610008+  43.83987352617628+ 36.60351143721994 + 33.56941738265945+ 33.97657594388816 + 37.2134755892125 + 41.66932957787554+  44.86468165542625+ 0 +0 +0 +0 +0 +0+ 0+ 0+ 0+0 +0 +0+ 0) plus the monomorphic value, which is 46.89212713

**Comment by xiaoming-liu** [comment]
That is right, based on the numbers you provided. However, the data looks
weird as you have so few monomorphic sites.   Are they sequence data?
Please make sure you include both types of monomorphic sites, that is,
those without any mutations and those fixed with mutations.
